jina/flow/base.py

```diff
@@ -1075,6 +1075,14 @@ def start(self):
 
         self._build_level = FlowBuildLevel.RUNNING
 
+        if (
+            self.args.infrastructure == InfrastructureType.K8S
+            and self.args.k8s_startup_time > 0
```

Comment:
what is this argument?

Reply:
The time to wait between the startup of the flow and the first request sent